src/Symfony/Component/Console/Question/Question.php

```diff
@@ -220,7 +215,7 @@ public function getValidator()
      *
      * @throws InvalidArgumentException in case the number of attempts is invalid
      */
-    public function setMaxAttempts($attempts)
+    public function setMaxAttempts(?int $attempts)
```

Comment:
you could remove the param doc

Reply:
@derrabus the now merged https://github.com/symfony/symfony/issues/33150 could help to clean those automatically